lib/development/dependencies.properties

```diff
@@ -22,7 +22,7 @@ dtdparser=com.liferay:com.wutka.dtd:1.20
 dumbster=com.liferay:com.dumbster.smtp:02856255a6cc7f
 error-prone-ant=com.google.errorprone:error_prone_ant:1.0.8
 freemarker=com.liferay:org.freemarker:2.3.29.LIFERAY-PATCHED-2
-gson=com.google.code.gson:gson:2.2.4
+gson=com.google.code.gson:gson:2.9.0
```

Comment:
@jeffwu0724 There is already a shared dependency of this jar, we should remove this.